src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/Pages/Counter.razor

```diff
@@ -0,0 +1,27 @@
+﻿@page "/counter"
+@*#if (UseServer && !UseClient) -->
```

Comment:
So, if you don't have Server or WebAssembly enabled, then the counter functionality will be broken, right? Should we disable the button if no interactivity is enabled?

Reply:
I'm actually going to remove the file from the template when no server or webassembly are present